__tests__/api.test.js

```diff
@@ -0,0 +1,34 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("GET:200 responds with all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const topicsObject = JSON.parse(res.text);
```

Comment:
Is there a benefit to using this? You will be able to access the body of the response on the body property and this won't require you to parse it.